Test/FMI3/fmi3_import_bouncingball_xml_test.cpp

```diff
@@ -228,7 +229,7 @@ void printVariableInfo(fmi3_import_t* fmu,
             break;
         }
         case fmi3_base_type_enum:{
-            printf("start = %lld\n", fmi3_import_get_enum_variable_start(fmi3_import_get_variable_as_enum(v)));
+            printf("start = %" PRId64 "\n", fmi3_import_get_enum_variable_start(fmi3_import_get_variable_as_enum(v)));
```

Comment:
```suggestion
            // doesn't work on mingw_64, but there are enums in bouncing ball
            // printf("start = %" PRId64 "\n", fmi3_import_get_enum_variable_start(fmi3_import_get_variable_as_enum(v)));
```

Not worth fixing just for mingw_64. These tests are largely conversions from fmi1 & fmi2 and were manually tested/inspected. Plus there are no enums in bouncingBall; so this part of the code is not executed either way.